src/main/scala/viper/gobra/frontend/info/implementation/property/Addressability.scala

```diff
@@ -102,4 +102,17 @@ trait Addressability extends BaseProperty { this: TypeInfoImpl =>
       case c: Constant => AddrMod.constant
       case e => Violation.violation(s"Expected variable, but got $e")
     }}
+
+  /** a parameter can be used as shared if it is included in the shared clause of the enclosing function or method */
+  lazy val isParameterUsedAsShared: PParameter => Boolean =
```

Comment:
would `isParameterShared` be better?

Reply:
I was unsure about the name. A parameter cannot be shared, but you can declare a shared copy of it for the loop. That is why I would be against `isParameterShared`, but I am open for other suggestions.

Reply:
I see. `isParameterUsedAsShared` sounds for me more like that it is indeed used as shared (more than just being declared as such). Hence, my favorite so far would be `canParameterBeUsedAsShared`

Reply:
I will change it to that.